README.md

```diff
@@ -243,23 +243,23 @@ rokuDeploy.createSignedPackage({
-- `[keyUp()](https://github.com/rokucommunity/roku-deploy/blob/v4/src/RokuDeploy.ts#L187)`
-- `[keyDown()](https://github.com/rokucommunity/roku-deploy/blob/v4/src/RokuDeploy.ts#L194)`
-- `[sendText()](https://github.com/rokucommunity/roku-deploy/blob/v4/src/RokuDeploy.ts#L201)`
-- `[closeChannel()](https://github.com/rokucommunity/roku-deploy/blob/v4/src/RokuDeploy.ts#L226)`
-- `[sideload()](https://github.com/rokucommunity/roku-deploy/blob/v4/src/RokuDeploy.ts#L239)`
-- `[rekeyDevice()](https://github.com/rokucommunity/roku-deploy/blob/v4/src/RokuDeploy.ts#L379)`
-- `[createSignedPackage()](https://github.com/rokucommunity/roku-deploy/blob/v4/src/RokuDeploy.ts#L426)`
-- `[deleteDevChannel()](https://github.com/rokucommunity/roku-deploy/blob/v4/src/RokuDeploy.ts#L614)`
-- `[captureScreenshot()](https://github.com/rokucommunity/roku-deploy/blob/v4/src/RokuDeploy.ts#L629)`
-- `[getOptions()](https://github.com/rokucommunity/roku-deploy/blob/v4/src/RokuDeploy.ts#L698)`
-- `[checkRequiredOptions()](https://github.com/rokucommunity/roku-deploy/blob/v4/src/RokuDeploy.ts#L738)`
-- `[getDeviceInfo()](https://github.com/rokucommunity/roku-deploy/blob/v4/src/RokuDeploy.ts#L787)`
-- `[getDevId()](https://github.com/rokucommunity/roku-deploy/blob/v4/src/RokuDeploy.ts#L854)`
+- [stage()](https://github.com/rokucommunity/roku-deploy/blob/v4/src/RokuDeploy.ts#L31)
+- [zip()](https://github.com/rokucommunity/roku-deploy/blob/v4/src/RokuDeploy.ts#L68)
+- [getFilePaths()]{https://github.com/rokucommunity/roku-deploy/blob/v4/src/RokuDeploy.ts#L124}
+- [keyPress()](https://github.com/rokucommunity/roku-deploy/blob/v4/src/RokuDeploy.ts#L179)
+- [keyUp()](https://github.com/rokucommunity/roku-deploy/blob/v4/src/RokuDeploy.ts#L187)
+- [keyDown()](https://github.com/rokucommunity/roku-deploy/blob/v4/src/RokuDeploy.ts#L194)
+- [sendText()](https://github.com/rokucommunity/roku-deploy/blob/v4/src/RokuDeploy.ts#L201)
+- [closeChannel()](https://github.com/rokucommunity/roku-deploy/blob/v4/src/RokuDeploy.ts#L226)
+- [sideload()](https://github.com/rokucommunity/roku-deploy/blob/v4/src/RokuDeploy.ts#L239)
+- [rekeyDevice()](https://github.com/rokucommunity/roku-deploy/blob/v4/src/RokuDeploy.ts#L379)
+- [createSignedPackage()](https://github.com/rokucommunity/roku-deploy/blob/v4/src/RokuDeploy.ts#L426)
+- [deleteDevChannel()](https://github.com/rokucommunity/roku-deploy/blob/v4/src/RokuDeploy.ts#L614)
+- [captureScreenshot()](https://github.com/rokucommunity/roku-deploy/blob/v4/src/RokuDeploy.ts#L629)
+- [getOptions()](https://github.com/rokucommunity/roku-deploy/blob/v4/src/RokuDeploy.ts#L698)
+- [checkRequiredOptions()](https://github.com/rokucommunity/roku-deploy/blob/v4/src/RokuDeploy.ts#L738)
+- [getDeviceInfo()](https://github.com/rokucommunity/roku-deploy/blob/v4/src/RokuDeploy.ts#L787)
+- [getDevId()](https://github.com/rokucommunity/roku-deploy/blob/v4/src/RokuDeploy.ts#L854)
```

Comment:
So, I was more thinking you would add a single link to `RokuDeploy.ts`, becuase these line numbers will change as we update the code. I think you can just list the functions without links, with a single link at the top saying "look through this file to see all the public methods available"

Reply:
Ohhh ok. Yeah the line numbers would change so one link sounds easier to maintain

Reply:
Done and also changed think to typescript interface for Roku Deploy Options